moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/current_state_monitor.h

```diff
@@ -56,6 +57,53 @@ class CurrentStateMonitor
   using TFConnection = boost::signals2::connection;
 
 public:
+  /**
+   * @brief      This class contains the rcl interfaces for easier testing
+   */
+  class RclInterface
```

Comment:
Should there be a better name? 
```suggestion
  class RclTestInterface
```

Reply:
This is just for testing right?

Reply:
This class seems like a generic tool that should not be specific to the CurrentStateMonitor. Maybe it should live in moveit_core/utils ?